src/modules/logicModule/interfaces/ILM_PC_PaymentRouter_v1.sol

```diff
@@ -7,14 +7,17 @@ interface ILM_PC_PaymentRouter_v1 {
     /// @param recipient The address that will receive the payment.
     /// @param paymentToken The token in which to pay.
     /// @param amount The amount of tokens the payment consists of.
-    /// @param dueTo The timestamp at which the payment SHOULD be fulfilled.
+    /// @param end The timestamp at which the payment SHOULD be fulfilled.
     function pushPayment(
         address recipient,
         address paymentToken,
         uint amount,
-        uint dueTo
+        uint start,
+        uint cliff,
+        uint end
     ) external;
 
+    /*
```

Comment:
Can this be cleaned up if the commented-out version is not required anymore? Looks like a leftover to me at least?